playbooks/os_updates.yml

```diff
@@ -32,16 +32,21 @@
       register: yum_update_status
       when: ansible_os_family == "RedHat"
 
-    - name: RedHat | remove unneeded packages
+    - name: RedHat | clear local repository
       ansible.builtin.yum:
-        autoremove: true
+        clean: true
       when: ansible_os_family == "RedHat"
 
     - name: Ubuntu | remove dependencies that are no longer required
       ansible.builtin.apt:
         autoremove: true
       when: ansible_os_family == "Debian"
 
+    - name: Ubuntu | remove dependencies that are no longer required
+      ansible.builtin.apt:
+        autoremove: true
+      when: ansible_os_family == "Debian"
+
```

Comment:
This looks like a duplicate of the task above. Are `autoremove` and `clean` the same thing?

Reply:
`autoremove` automatically removes dangling software `clean` clears `var/cache/apt/archives/` and `/var/cache/apt/archives/partial/`

Reply:
Yeah, I meant in ansible-speak. Line 42 already says `autoremove`. I'm thinking line 47 should say `clean`?

Reply:
🤦🏿‍♂️